apps/organisations/forms.py

```diff
@@ -175,6 +171,14 @@ def __init__(self, *args, **kwargs):
             for name, field_cls, kwargs in self.translated_fields:
                 self.instance.set_current_language(lang_code)
                 field = field_cls(**kwargs)
+                # The CKEditor5Widget doesn't have a label field, so we need to set it
+                # after creating the object
+                if isinstance(CKEditor5Widget, field):
```

Comment:
ups, this should be `isinstance(field, CKEditor5Widget)` I think, right? :/

Reply:
yes i fixed that